src/bin-install.ts

```diff
@@ -14,15 +14,21 @@ typings install [<name>=]<location>
   <location>  The location to read from (described below)
 
 Valid Locations:
-  [<source>!]<pkg>[@<version>][#<tag>]
+  <pkg>[@<version>][#<tag>]
+    Install from "defaultSource" specified in `.typingsrc`
+      default to "npm" for non-ambient package
+      default to "dt" for ambient package
+  <source>!<pkg>[@<version>][#<tag>]
+    Install from registry mirror or DefinitelyTyped (see <source> below)
```

Comment:
Sure, I agree. I just don't want to add a paragraph of explanation to every piece - this'll be very long, so I'm hesitant to start. Basically, if I can shrink this, I want to. I'll open another PR.